src/test/java/com/kodekonveyor/repo/backend/api/lookup/LookupRepositoryByNameServiceLookupRepositoryByNameTest.java

```diff
@@ -0,0 +1,108 @@
+package com.kodekonveyor.repo.backend.api.lookup;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import com.kodekonveyor.annotations.TestedBehaviour;
+import com.kodekonveyor.annotations.TestedService;
+import com.kodekonveyor.exception.ThrowableTester;
+import com.kodekonveyor.repo.api.LerpoiTestData;
+import com.kodekonveyor.repo.api.SumtiDTOTestData;
+import com.kodekonveyor.repo.api.SumtiEntityRepositoryStubs;
+import com.kodekonveyor.repo.backend.api.LerpoiEntityRepositoryStubs;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@RunWith(MockitoJUnitRunner.class)
+@TestedBehaviour("lookup repository by name")
+@TestedService("LookupRepositoryByNameService")
+public class LookupRepositoryByNameServiceLookupRepositoryByNameTest
+    extends LookupRepositoryByNameServiceTestBase {
+
+  @Test
+  @DisplayName(
+    "looks up the sumti which is a repository, with the lerpoi given"
+  )
+  void test() {
+    LerpoiEntityRepositoryStubs.behaviour(lerpoiEntityRepository);
+    SumtiEntityRepositoryStubs.behaviour(sumtiEntityRepository);
```

Comment:
Call the behaviour method from the setup method of the LookupRepositoryByNameServiceTestBase.